### PR TITLE
Initial split

### PR DESCRIPTION
**Pull request by CoMPaTech**
Following https://github.com/plugwise/python-plugwise/pull/287 and https://github.com/plugwise/python-plugwise/pull/295 after forking this repository and https://github.com/plugwise/plugwise-beta/pull/384 to complete the circle

### CODE REVIEW

File: custom_components/plugwise_usb/models.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+from homeassistant.helpers.entity import EntityDescription
+
+from .const import STICK, USB_MOTION_ID, USB_RELAY_ID
+
+
+@dataclass
+class PlugwiseRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    plugwise_api: str
```
Comment (bouwew) [inline custom_components/plugwise_usb/models.py:32]: This part is no longer needed after the split.

Also below all `plugwise_api=STICK,` lines can be removed.

File: custom_components/plugwise_usb/models.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+
+@dataclass
+class PlugwiseEntityDescription(EntityDescription, PlugwiseRequiredKeysMixin):
```
Comment (bouwew) [inline custom_components/plugwise_usb/models.py:36]: class PlugwiseEntityDescription(EntityDescription):

Comment (bouwew) [reply custom_components/plugwise_usb/models.py:36]: Or remove this class and rename below classes to SensorEntityDescription, etc.

Maybe even better, move all descriptions (back) into the relevant files, as done in Core Plugwise.

File: custom_components/plugwise_usb/strings.json
Comment (bouwew) [inline custom_components/plugwise_usb/strings.json:1]: All translations are related to Anna/Adam/P1, these can all be removed.

Comment (bouwew) [reply custom_components/plugwise_usb/strings.json:1]: Also, other stuff can be removed because the Config Flow is much simpler now.

Comment (bouwew) [reply custom_components/plugwise_usb/strings.json:1]: And, don't forget to update the translation-files.

File: custom_components/plugwise_usb/models.py (hunk 1 above)
Comment (CoMPaTech) [reply custom_components/plugwise_usb/models.py:32]: Tracking via #3

File: custom_components/plugwise_usb/models.py (hunk 2 above)
Comment (CoMPaTech) [reply custom_components/plugwise_usb/models.py:36]: Tracking via https://github.com/plugwise/plugwise_usb-beta/issues/3

File: custom_components/plugwise_usb/strings.json
Comment (CoMPaTech) [reply custom_components/plugwise_usb/strings.json:1]: Tracking via #4

**Comment by CoMPaTech** [comment]
Lets review this and bump to version for release so HACS better detects/distinguishes between both beta's

**Pull request merged**